### PR TITLE
lvremove: add page

### PR DESCRIPTION
**Pull request by tkiatd**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/linux/lvremove.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# lvremove
+
+> Remove one or more logical volumes.
+
+- Remove a logical volume in a volume group:
+
+`sudo lvremove {{volume_group}}/{{logical_volume}}`
```
Comment (navarroaxel) [inline pages/linux/lvremove.md:7]: We don't include `sudo` in the examples.
```suggestion
`lvremove {{volume_group}}/{{logical_volume}}`
```

File: pages/linux/lvremove.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+- Remove all logical volumes in a volume group:
+
+`sudo lvremove {{volume_group}}`
```
Comment (navarroaxel) [inline pages/linux/lvremove.md:11]:
```suggestion
`lvremove {{volume_group}}`
```

File: pages/linux/lvremove.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# lvremove
+
+> Remove one or more logical volumes.
```
Comment (navarroaxel) [inline pages/linux/lvremove.md:3]: We can add the link to the `lvm2` site.
```suggestion
> Remove one or more logical volumes.
> More information: <https://sourceware.org/lvm2/>.
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hey @tkiatd, thanks for your submission. I left some comments.

File: pages/linux/lvremove.md (hunk 1 above)
Comment (sbrl) [reply pages/linux/lvremove.md:7]: I think it should stay here, @navarroaxel. Generally we include a `sudo` prefix if a command requires it to function.

File: pages/linux/lvremove.md (hunk 2 above)
Comment (sbrl) [reply pages/linux/lvremove.md:11]: I think it should stay here, @navarroaxel. Generally we include a `sudo` prefix if a command requires it to function.

File: pages/linux/lvremove.md (hunk 3 above)
Comment (tkiatd) [reply pages/linux/lvremove.md:3]: Hi @navarroaxel, thanks for your comments. I modified it. Cheers!

**Review by navarroaxel — APPROVED** [review]
Thanks @tkiatd, awesome!

**Review by sbrl — APPROVED** [review]
Nice one, @tkiatd! Documenting lvm is a great idea. I for one certainly find it easier to use tools like this when there are tldr pages there to help :P

**Pull request merged**